packages/nx/src/command-line/logout/command-object.ts

```diff
@@ -0,0 +1,12 @@
+import { CommandModule } from 'yargs';
+import { withVerbose } from '../../command-line/yargs-utils/shared-options';
+
+export const yargsLogoutCommand: CommandModule = {
+  command: 'logout',
+  describe:
+    'Logout from Nx Cloud by removing a personal access token from your local machine. The personal access token will be revoked from Nx Cloud and you will need to login again to regain access to Nx Cloud.',
```

Comment:
This seems like too much information about the implementation details.

I think we can shorten it to..

```suggestion
    'Logout from Nx Cloud',
```